mysql-test/suite/galera_encryption/t/wscache_encryption_on_off.test

```diff
@@ -4,6 +4,7 @@
 
 --source include/galera_cluster.inc
 --source include/count_sessions.inc
+--source include/have_debug_sync.inc
```

Comment:
This is already present in 8.0 trunk. 
https://github.com/percona/percona-xtradb-cluster/commit/25f81ea441e2d1a7422492b6c98063388749956a

Reply:
yup, but we need it here as well :)